layout/theme.liquid

```diff
@@ -123,7 +123,7 @@
           </div>
         </div>
         {% else %}
-          {% unless current_user.is_anonymous? %}
+          {% if current_user.is_wholesale? %}

```

Comment:
@etagwerker I don't think the last check for `current_user.is_admin?` is necessary, we are already checking it in the `if` above it, before entering this `else` condition. See: https://github.com/ombushop/wooden/blob/bugfix/store-messages/layout/theme.liquid#L118-L134